File: Employee.java
class Employee {
  
    private String name, lastName, birthdate, admissionDate, rfc, curp, rol;
    private String [][] schedule;
    private double salary;
    private int id;
    
    public Employee(String name, String lastName, String birthdate, String dateAdmission, String rfc, String curp, double salary, String [][] schedule, String rol, int id) {
        this.name = name;
        this.lastName = lastName;
        this.birthdate = birthdate;
        this.admissionDate = dateAdmission;
        this.rfc = rfc;
        this.curp = curp;
        this.salary = salary;
        this.schedule = schedule;
        this.rol = rol;
        this.id = id;
    }

    String getName() {
        return name;
    }

    String getLastName() {
        return lastName;
    }

    int getId() { //obtiene id
        return id;
    }

    String getRol() {
        return rol;
    }

    String [][] getSchedule() {
        return schedule;
    }

    void printSchedule() {
        System.out.println("\n---- Horario ----\n");
        for(int x = 0; x < 7; x++) {
            System.out.println(schedule[0][x] + ": " + schedule[1][x]);
        }
    }

    void setId(int id) {
        this.id = id;
    }

    void setName(String name) {
        this.name = name;
    }

    void setLastName(String lastName) {
        this.lastName = lastName;
    }

    void setBirthdate(String birthdate) {
        this.birthdate = birthdate;
    }

    void setAdmissionDate(String admissionDate) {
        this.admissionDate = admissionDate;
    }

    void setRfc(String rfc) {
        this.rfc = rfc;
    }

    void setCurp(String curp) {
        this.curp = curp;
    }

    void setSalary(double salary) {
        this.salary = salary;
    }

    void setSchedule(String[][] schedule) {
        this.schedule = schedule;
    }

    void setRol(String rol) {
        this.rol = rol;
    }

    String employeeGetData() { //datos de empleado 
        return String.format("Nombre: %s %s \nID: %d \nFecha de nacimiento: %s \nFecha de ingreso: %s \nRFC: %s \nCURP: %s \nSalario: %.2f MX \nRol: %s",
            this.name, this.lastName, this.id, this.birthdate, this.admissionDate, this.rfc, this.curp, this.salary, this.rol);
    }
}
    

File: Zoo python/Employee.py
class Employee:
    def __init__(self, name, last_name, birthdate, admission_date, rfc, curp, salary, schedule, rol, id):
        self.name = name
        self.last_name = last_name
        self.birthdate = birthdate
        self.admission_date = admission_date
        self.rfc = rfc
        self.curp = curp
        self.salary = salary
        self.schedule = schedule
        self.rol = rol
        self.id = id

    def get_name(self):
        return self.name

    def get_last_name(self):
        return self.last_name

    def get_id(self):
        return self.id

    def get_rol(self):
        return self.rol

    def get_schedule(self):
        return self.schedule

    def print_schedule(self):
        print("\n---- Horario ----\n")
        for x in range(7):
            print(self.schedule[0][x] + ": " + self.schedule[1][x])

    def set_id(self, id):
        self.id = id

    def set_name(self, name):
        self.name = name

    def set_last_name(self, last_name):
        self.last_name = last_name

    def set_birthdate(self, birthdate):
        self.birthdate = birthdate

    def set_admission_date(self, admission_date):
        self.admission_date = admission_date

    def set_rfc(self, rfc):
        self.rfc = rfc

    def set_curp(self, curp):
        self.curp = curp

    def set_salary(self, salary):
        self.salary = salary

    def set_schedule(self, schedule):
        self.schedule = schedule

    def set_rol(self, rol):
        self.rol = rol

    def employee_get_data(self):
        return f"Nombre: {self.name} {self.last_name} \nID: {self.id} \nFecha de nacimiento: {self.birthdate} \nFecha de ingreso: {self.admission_date} \nRFC: {self.rfc} \nCURP: {self.curp} \nSalario: {self.salary:.2f} MX \nRol: {self.rol}"
